tests/ethernet/test_ethernet_network.py

```diff
@@ -92,14 +91,12 @@ def test_load_firmware_no_connection():
 @pytest.mark.skip()
 @pytest.mark.no_connection()
 def test_load_firmware_wrong_user_pwd():
-    # TODO: implement
```

Comment:
Create issues for the pending work of these tests

Reply:
The issue has already been created (INGK-1025).